frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -141,8 +146,14 @@ const Tooltip = () => {
     matchingEntries,
     matchingEntities,
     dateRange,
+    parent,
   } = additionalInfos;
 
+  const hasChild = useMemo(() => {
+    return parent !== label && parent;
+  }, [parent, label]);
+
+  if (!displayTooltip) return <ActivateTooltip />;
```

Comment:
For early returns, please give it some space
```suggestion
  if (!displayTooltip) return <ActivateTooltip />;
  
```